pulsar-broker/src/test/java/org/apache/pulsar/schema/compatibility/SchemaCompatibilityCheckTest.java

```diff
@@ -267,6 +269,23 @@ public void testIsAutoUpdateSchema(SchemaCompatibilityStrategy schemaCompatibili
         assertEquals(personTwo.id, 2);
         assertEquals(personTwo.name, "Lucy");
 
+        producer.close();
+        consumerTwo.close();
+
+        producer = producerThreeBuilder.create();
+        consumerTwo = comsumerBuilder.subscribe();
+
+        admin.namespaces().setIsAllowAutoUpdateSchema(namespaceName.toString(), false);
```

Comment:
Set schema auto update strategy need before producer creation, because we do the schema compatibility check when create a new producer or consumer.